src/main/java/binaries/maday/BinaryArray.java

```diff
@@ -18,6 +18,14 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        //throw new UnsupportedOperationException("Not supported yet.");
+        int potencia = binaries.length-1;
```

Comment:
Let's use english as standard language for names. Also, missing space before and after `-`